websockets/conversation.go

```diff
@@ -69,25 +120,38 @@ func (c *Conversation) Run() {
 				return
 			}
 
-			patches, err := dmp.PatchFromText(string(message.content))
+			if _, ok := c.clients[message.sender]; !ok {
+				continue
+			}
+
+			update := models.Update{}
+			if err := json.Unmarshal(message.content, &update); err != nil {
+				log.Printf("Failed to parse WebSocket message content: %v", err)
+				c.deleteClient(message.sender)
+				continue
+			}
+
+			originalVersion := update.Version
+			ok, err := c.processUpdate(&update)
```

Comment:
The `ok` boolean should be checked at some point right? Or line 70 should be changed to return `false, err` instead of `false, nil`.

Reply:
Oh weird, yeah I just forgot to handle it.